doc/classes/AnimatedSprite3D.xml

```diff
@@ -83,12 +85,16 @@
 	</methods>
 	<members>
 		<member name="animation" type="String" setter="set_animation" getter="get_animation">
+			The current animation from the [code]frames[/code] resource. If this value is changed, the [code]frame[/code] counter is reset.
 		</member>
 		<member name="frame" type="int" setter="set_frame" getter="get_frame">
+			The current frame index.
```

Comment:
Could be clearer. Suggestion: `The index of the animation frame the node displays/displayed on screen.`
NB: passive voice is okay in some instances - when the node is acted upon, here processed by the renderer.